noether/CMakeLists.txt

```diff
@@ -45,16 +45,16 @@ catkin_package(
     mesh_segmenter
   CATKIN_DEPENDS
     actionlib
-    mesh_segmenter
     noether_conversions
     noether_msgs
-    path_sequence_planner
     pcl_conversions
     roscpp
-    tool_path_planner
   DEPENDS
-    VTK
+    mesh_segmenter
+    path_sequence_planner
     PCL
+    tool_path_planner
```

Comment:
Only packages that export the variables `xxx_INCLUDE_DIRS` and `xxx_LIBRARIES` should go in the `DEPENDS` section. For pure CMake packages that export targets like the ones you're making, you should use `CFG_EXTRAS` ([example](https://github.com/marip8/tesseract_ros/blob/ad06c70bdb7a464d827bce7ff738598a9ec642b9/tesseract_ros/tesseract_rviz/CMakeLists.txt#L72-L73))and make a `.cmake` file that calls `find_package` on those packages. ([example](https://github.com/marip8/tesseract_ros/blob/ad06c70bdb7a464d827bce7ff738598a9ec642b9/tesseract_ros/tesseract_rviz/cmake/tesseract_rviz-extras.cmake))